tests/returntypes/transforms/test_mitigate_new.py

```diff
@@ -0,0 +1,570 @@
+# Copyright 2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for mitigation transforms.
+"""
+# pylint:disable=no-self-use
+import pytest
+
+from packaging import version
+
+import pennylane as qml
+from pennylane import numpy as np
+from pennylane.tape import QuantumTape
+from pennylane.transforms import (
+    mitigate_with_zne,
+    poly_extrapolate,
+    richardson_extrapolate,
+    fold_global,
+)
+
+with QuantumTape() as tape:
+    qml.BasisState([1], wires=0)
+    qml.RX(0.9, wires=0)
+    qml.RY(0.4, wires=1)
+    qml.CNOT(wires=[0, 1])
+    qml.RY(0.5, wires=0)
+    qml.RX(0.6, wires=1)
+    qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+
+with QuantumTape() as tape_base:
+    qml.RX(0.9, wires=0)
+    qml.RY(0.4, wires=1)
+    qml.CNOT(wires=[0, 1])
+    qml.RY(0.5, wires=0)
+    qml.RX(0.6, wires=1)
+
+with QuantumTape() as tape_multi:
```

Comment:
It seems like this tape is not being used at all.

Reply:
Removed it.